### ISSUE TITLE
Current commit breaks script

### ISSUE THREAD

**Issue by bb-eric** [Torantulino/Auto-GPT#481]
### Duplicates

- [X] I have searched the existing issues

### Steps to reproduce 🕹

```
$ python scripts/main.py 
Traceback (most recent call last):
  File "/Auto-GPT/scripts/main.py", line 3, in <module>
    import commands as cmd
  File "/Auto-GPT/scripts/commands.py", line 12, in <module>
    from image_gen import generate_image
  File "/Auto-GPT/scripts/image_gen.py", line 4, in <module>
    from PIL import Image
ModuleNotFoundError: No module named 'PIL'
```

using Python 3.11.2

### Current behavior 😯

Won't run, previous commits worked well (I tested "cc05139843853ab2ac091c38253a810e418e675c" and it worked)
Now, with "7e108d8ab37d828c3d7b129399d8d9927154f801" won't run.

I guess you need to update the requirements and add Pillow.

**Comment by richbeales** [comment]
Verified, requirements.txt needs updating.

**Comment by richbeales** [comment]
**pip install pillow** resolves this

**Comment by Torantulino** [comment]
Fixed in #482
